examples/modal.js

```diff
@@ -1,11 +1,14 @@
 goog.provide('app.modal');
 
 /** @suppress {extraRequire} */
-goog.require('ngeo.modalDirective');
+goog.require('ngeo.message.modalComponent');
 
 
 /** @type {!angular.Module} **/
-app.module = angular.module('app', ['ngeo']);
+app.module = angular.module('app', [
+  'ngeo',
```

Comment:
`ngeo.module.name` is better no ?
Look also other ngeo example (`git grep \'ngeo\'`)

Reply:
This should be all of them.